conda/gateways/disk/create.py

```diff
@@ -36,6 +36,14 @@
     exit(%(func)s())
 """)
 
+private_pkg_entry_point_template = dals("""
+import os
+import sys
+if __name__ == '__main__':
+    exec_path = os.path.join("%(private_env_path)s", "bin", "%(app_name)s")
```

Comment:
Don't use `os.path.` here.  Just `join`.

Also, is `bin` correct?  I'm not sure...

Actually, the full `exec_path` could/should probably come from `CreateApplicationEntryPointAction` as `source_full_path`.